### ISSUE TITLE
Suggestion - fix YouTube tag code in Master... 

### ISSUE THREAD

**Issue by JP58** [YAFNET/YAFNET#138]
Since many now use https, the YouTube bbcode has to be fixed in the master in my opinion. 

See Search Regex box in screenshot for both http & https support..

I myself would prefer the embed code to be https by default, so it can also be used in true https sites...  See the Replace Regex box in the screenshot... 

![image](https://cloud.githubusercontent.com/assets/5246746/2906515/03702a62-d612-11e3-8901-6a0298427e2b.png)

SInce I don't know how to do code markup in the Github editor, I hope the screenshot is enough. 
Basically replace http with http(s)? , only in 2 places in the Search RegEx - and http with https in the Replace RegEx...

Since YouTube (Google) links with https if you are logged in to your Google account, I think it should be standard procedure. 

In fact, I think all URL related bbcodes should have this option...   :-)
